app/presenters/rtl_show_presenter.rb

```diff
@@ -75,7 +75,11 @@ def override_title_field
   end
 
   def html_title
-    super.split("<br />").map(&:html_safe).join(", ").gsub(/<.*?>/, "")
+    # Insert field_value_separator between tags
+    value = super.gsub("li><li", "li>#{field_value_separator}<li")
```

Comment:
Maybe we can split on the full tag like
`value.split("</li><li>").map(&:html_safe).join(", ").gsub(/<.*?>/, "")`